charts/opentelemetry-demo/examples/bring-your-own-observability/values.yaml

```diff
@@ -1,11 +1,24 @@
-
-default:
-  envOverrides:
-    - name: OTEL_COLLECTOR_NAME
-      value: my-otel-collector.opentelemetry-ns
-
-opentelemetry-collector:
-  enabled: false
+opentelemetry-kube-stack:
+  collectors:
+    daemon:
+      config:
+        exporters:
+          otlphttp/bring-your-own-o11y:
```

Comment:
@julianocosta89 @puckpuck can you please check this "bing your own o11y" solution works for the use cases that come to your mind?